firmware/ubirch/m66/m66_tcp.h

```diff
@@ -0,0 +1,46 @@
+/*!
+ * @file
+ * @brief M66 TCP/IP operations.
+ *
+ * @author Niranjan H. Rao
+ * @date 2016-10-07
+ *
+ * @copyright &copy; 2015 ubirch GmbH (https://ubirch.com)
+ *
+ * ```
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ```
+ */
+
+#ifndef UBIRCH_FIRMWARE_MQTT_H
+#define UBIRCH_FIRMWARE_MQTT_H
+
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// TODO Make the read buffer size configurable
+#ifndef MQTT_READ_BUFFER
+  #define MQTT_READ_BUFFER 255
+#endif
+
+bool modem_mqtt_connect(const char *apn, const char *user, const char *password, uint32_t timeout);
```

Comment:
This is confusing: m66_tcp.c, but inside is modem_mqtt_connect. Rename the functions to modem_tcp*